Earthfile

```diff
@@ -462,21 +462,35 @@ INIT:
     IF ! __have_command __have_command
         COPY --chmod=755 tools/__tool /usr/local/bin/__tool
         RUN __tool __init
-        IF __can_install epel-release
-            RUN __do "Enabling EPEL repositories..." __install epel-release
-        END
-        IF ! __have_command curl
-            RUN __do "Installing curl..." __install curl
-        END
+    END
 
-        ARG uv_version = "0.8.15"
-        ARG uv_install_sh_url = "https://astral.sh/uv/$uv_version/install.sh"
-        IF ! __have_command uv
-            RUN curl -LsSf "$uv_install_sh_url" \
-                    | env UV_UNMANAGED_INSTALL=/opt/uv sh - \
-                && __alias uv  /opt/uv/uv \
-                && __alias uvx /opt/uv/uvx
-        END
+    IF __can_install epel-release
+        # Installing epel-release must happen separately since it can update the
+        # available repositories on certain systems
+        RUN __do "Enabling EPEL repositories..." __install epel-release
+    END
+
+    # Ensure that we have Tar, Curl, and Gzip
+    LET pkgs = ""
+    IF ! __have_command tar
+        SET pkgs = $pkgs tar
+    END
+    IF ! __have_command curl
+        SET pkgs = $pkgs curl
+    END
+    IF ! __have_command gzip
+        SET pkgs = $pkgs gzip
+    END
+    RUN __do "Installing basic packages ($pkgs)" __install $pkgs
+
+    # Ensure that we have UV
+    ARG uv_version = "0.8.15"
+    ARG uv_install_sh_url = "https://astral.sh/uv/$uv_version/install.sh"
+    IF ! __have_command uv
+        RUN curl -LsSf "$uv_install_sh_url" \
```

Comment:
Even if not for the source tarballs downloaded by the install script, recommend adding checksum validation for the install script itself.